Makefile.docker

```diff
@@ -11,13 +11,41 @@ REPO ?= ${DOCKER_USER}/${DOCKER_REPO}
 # latest == LTS
 # rolling == Latest release
 # devel == Latest development release
-UBUNTU_TAGS := latest rolling devel artful zesty yakkety xenial trusty
+UBUNTU_TAGS := latest rolling devel bionic artful zesty yakkety xenial trusty
 TAGS ?= $(UBUNTU_TAGS)
 
 BUILD_TAG_PREFIX ?= build-
 
 all: build
 
+##
+## Host build deps
```

Comment:
Building prefixout for the host (allowing for cross-compilation is why it's separate from that in Makefile.docker) in a CircleCI-compatible way.

You can see the benefits of using prefixout in the build output. You can now see during parallel compilation of N images which output is from which. It's beautiful.